taipy/common/config/__init__.py

```diff
@@ -21,9 +21,9 @@
 !!! example "Standard usage"
```

Comment:
The previous lines should be changed as well replacing `taipy.config` by `taipy.common.config`